bin/node/runtime/src/lib.rs

```diff
@@ -171,8 +173,8 @@ impl frame_system::Trait for Runtime {
 	type BlockNumber = BlockNumber;
 	type Hash = Hash;
 	type Hashing = BlakeTwo256;
-	type AccountId = AccountId;
-	type Lookup = Indices;
+	type AccountId = AccountIdBr;
```

Comment:
Why do we need to use modified version with Signer and IdentifyAccount

Reply:
How can we make sure it will not break anything?

Reply:
To support Multisignature of chainbride. 
To make sure: 1. Auto UT 2. Manual test. Our current approach may be have oppsite direction. We tried to integrate chainbridge to PoS node. There are maybe better way such as: integrate PoS to chainbridge or understand the hold chainbridge then implement total new chain bridge that match to PoS Node.